background/test/testApp/package.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "background",
+    "private": true,
+    "description": "This directory contains an example of a Cloud Function.",
+    "main": "translate.js",
+    "author": "Google LLC",
+    "license": "Apache-2.0",
+    "scripts": {
+      "test": "mocha --exit test/*.test.js"
+    },
+    "dependencies": {
+      "@google-cloud/firestore": "^3.0.0",
+      "@google-cloud/translate": "^6.0.0"
+    },
+    "devDependencies": {}
```

Comment:
It's weird to me that this calls into mocha, but doesn't take a dev dependency on it

Reply:
That is also a mistake from copying over...good catch, thanks!

Reply:
Still looks to be empty 🤔

Reply:
I could have sworn I just deleted the mocha script... This folder doesn't get tested, so it shouldn't call on mocha at all. I'll re-delete!